fe/src/components/Input/PriceInput.tsx

```diff
@@ -0,0 +1,39 @@
+import { styled } from 'styled-components';
+import { Input } from './Input';
+import React from 'react';
+
+type Props = {
+  currencyUnit: 'won' | 'dollar';
```

Comment:
```suggestion
  currencyUnit: keyof typeof unit;
```
[제안]
이렇게 정의하면 unit 객체에 단위가 추가되어도 두 번 수정하지 않아도 되겠네요~

Reply:
keyof typeof으로 정의하면 말씀해주신 것처럼 훨씬 깔끔하고 유지보수에도 용이할 것 같네요!